pkg/apiserver/apiserver.go

```diff
@@ -168,6 +172,8 @@ func installAPIGroup(s *APIServer, c completedConfig) error {
 	cpv1beta2Storage["networkpolicies/status"] = networkPolicyStatusStorage
 	cpv1beta2Storage["nodestatssummaries"] = nodeStatsSummaryStorage
 	cpv1beta2Storage["groups"] = groupStorage
+	cpv1beta2Storage["groupassociation"] = groupAssociationStorage
+	cpv1beta2Storage["clustergroupmembership"] = groupMembershipStorage
```

Comment:
The API path normally has the same name as the struct, in plural. For instance, the struct should be called ClusterGroupMembers and the path should be clustergroupmembers?

Reply:
Thanks for the suggestion. Updated the API path names accordingly.